bentoml/_internal/log.py

```diff
@@ -47,15 +51,17 @@ class TraceFormatter(Formatter):
     """
 
     def __init__(self):
-        Formatter.__init__(self, fmt=TRACED_LOG_FORMAT, datefmt=DATE_FORMAT)
+        super().__init__(fmt=TRACED_LOG_FORMAT, datefmt=DATE_FORMAT)
         self.control_formatter = Formatter(SERVICE_LOG_FORMAT, datefmt=DATE_FORMAT)
-        self.trace_formatter = Formatter(TRACED_LOG_FORMAT, datefmt=DATE_FORMAT)
+
+    @property
+    def trace_formatter(self) -> Formatter:
```

Comment:
why do we need the refactoring here?  it's inconsistent with control_formatter now. let's reverse it back?